packages/transport-circuit-relay-v2/src/errors.ts

```diff
@@ -17,3 +17,27 @@ export class DurationLimitError extends Error {
     this.name = 'DurationLimitError'
   }
 }
+
+/**
+ * There were enough relay reservations already
+ */
+export class HadEnoughRelaysError extends Error {
+  static name: string = 'HadEnoughRelaysError'
+  name: string = 'HadEnoughRelaysError'
+}
+
+/**
+ * An attempt to open a relayed connection over a relayed connection was made
+ */
+export class DoubleRelayError extends Error {
+  static name: string = 'DoubleRelayError'
+  name: string = 'DoubleRelayError'
+}
+
+/**
+ * An attempt to open a relayed connection over a relayed connection was made
+ */
```

Comment:
It doesn't seem like this is used anywhere, but this is the same description as `DoubleRelayError`

```suggestion
/**
 * An attempt to open a relayed connection when we already have `reservationConcurrency` relay reservations
 */
```

Reply:
It's thrown when the relay reservation queue is full.